src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/jpa/VisitStatus.kt

```diff
@@ -6,8 +6,6 @@ enum class VisitStatus(
 ) {
   RESERVED("Reserved"),
   BOOKED("Booked"),
-  CANCELLED_BY_PRISONER("Cancelled by Prisoner"),
-  CANCELLED_BY_VISITOR("Cancelled by Visitor"),
-  CANCELLED_BY_PRISON("Cancelled by Prison"),
+  CANCELLED("Cancelled"),
   ATTENDED("Attended")
```

Comment:
Do you think we need the status 'Attended' and if so should we also have 'Non-Attended'?  Do you think this would be better as an outcome as per the migrated data.

Reply:
Makes sense I suppose, did not really think of that!

Will add :

- Attended
- Non-Attended